packages/css/src/components/spotlight/README.md

```diff
@@ -7,6 +7,10 @@ Emphasizes a section on a page through a distinctive background colour.
 ## Guidelines
 
 - Display the Spotlight at the entire width of the [Screen](/docs/components-layout-screen--docs); do not position it on the [Grid](/docs/components-layout-grid--docs).
+- Do not use other background colours than the options below.
+  The blue appearance of Footer must remain unique – this is why that colour is not an option for Spotlight.
+  Red, black and grey aren’t options either.
```

Comment:
Do we need to describe this? If we just don't offer it as an option, we already communicate this, no?

Reply:
Yes, removed.